en/hosting_guide/configuration.rst

```diff
@@ -14,20 +241,34 @@ Cron jobs on Open Orchestra
 
 Open Orchestra has 3 cron jobs created with the provisioning:
 
-* **Sitemap**: automatically generates sitemap files for every sites.
-* **Robots**: automatically generates the robots.txt files for every sites.
-* **ErrorPages**: automatically generates the special error pages files for every sites (eg 404 & 503 status).
+.. code-block::
+
+    0 2 * * * php /var/www/front-open-orchestra/current/app/console -e=prod orchestra:sitemaps:generate 2>> /tmp/cron.error.message
+    0 2 * * * php /var/www/front-open-orchestra/current/app/console -e=prod orchestra:robots:generate 2>> /tmp/cron.error.message
+    59 0-23 * * * if [ -s '/tmp/cron.error.message' ]; then  cat /tmp/cron.error.message | mailx -s "cron error" contact@open-orchestra.com; fi; rm /tmp/cron.error.message;
+
```

Comment:
Rather than a single block, I'd make 3 distincts blocks, each coming with a quick explanation of what the job does